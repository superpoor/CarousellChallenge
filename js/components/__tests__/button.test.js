import React from 'react';
import renderer from 'react-test-renderer';

import Button from '../button'

it('renders correct button component', () => {
  const rendered = renderer.create(
    <Button 
      backgroundColor="red"
      title="button"
      titleColor="white"
      onPress={() => {}}
      disabled
    />
  ).toJSON();
  expect(rendered).toMatchSnapshot();
});