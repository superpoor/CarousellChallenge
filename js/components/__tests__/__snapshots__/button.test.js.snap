// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct button component 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "red",
      "borderRadius": 3,
      "margin": 5,
      "opacity": 1,
      "paddingHorizontal": 20,
      "paddingVertical": 10,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 16,
          "fontWeight": "bold",
        },
        Object {
          "color": "white",
        },
      ]
    }
  >
    button
  </Text>
</View>
`;
